normalizer/app/geo_processor.py

```
@@ -35,131 +39,190 @@ def replacer(match):
-                rendered = render_template(template, row, cache, full_api_address, api_key or "", sleep_msec)
+                rendered = render_template(
+                    idx, template, row, cache, full_api_address, api_key, sleep_msec, gsi_check, gsi_distance, priority, log_callback
+                )
                 out_row.append(rendered)
-        
+
         results.append(out_row)
-    
+
+    if log_callback:
+        log_callback(f"完了")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)